src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -261,13 +258,26 @@ applyCoinbase v logger dbEnv (Miner mid mks) reward@(ParsedDecimal d) pd parentH
     forkTime = vuln797FixDate v
     fork1_3InEffect = blockTime >= forkTime
     throwCritical = fork1_3InEffect || enfCBFailure
+
     blockTime = blockTimeOf pd
+        -- FIXME it should hold that @blockTime == _blockCreationTime parentHeader@, but
+        -- in some unit test runs that is not the case
+
+    bh = fromIntegral $ _pdBlockHeight pd
+
+    cid = V._chainId parentHeader
+        -- FIXME it should hold that
+        -- @cid == unsafeFromText $ P._chainId $ _pmChainId $ _pdPublicMeta pd@
+        -- but in some unit test runs that is not the case. The chain id in public data is empty.
+
+    chash = Pact.Hash $ encodeToByteString $ _blockHash parentHeader
+        -- FIXME ceck that @ _pdPrevBlockHash pd == encode _blockHash@
+        -- FIXME chash includes the quoted text of the parent header. Is that correct?
```

Comment:
@emilypi do you have thoughts on these FIXME's? Can we do some cleanup here or add some clarifying comments, or just leave it as is?

Reply:
This fix me is corret

Reply:
I believe the change here addresses both FIXMEs doesn't it?